source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -2065,6 +2054,29 @@ configuration settings.
       This environment variable configuration setting the 
       :envvar:`MINIO_IDENTITY_LDAP_USER_DN_SEARCH_FILTER` environment variable.
 
+   .. mc-conf:: enabled
+      :delimiter: " "
+
+      *Optional*
+
+      Set to ``false`` to disable the AD/LDAP configuration.
+
+      Applications cannot generate STS credentials or otherwise authenticate to MinIO using the configured provider if set to ``false``.
```

Comment:
Maybe reverse this sentence to put `if set to false` at the beginning. This is about the setting first, then what applications may or may not do as a result. (Also in the OpenID section below.)

Reply:
Good catch and suggestion. I concur.

Reply:
Oooh good suggestion, taken.